### DISCUSSION TITLE
v1.1.28

### DISCUSSION THREAD

**Discussion by kirb** [wstyres/Zebra#1985] [Releases]
* Fixes Zebra crashing on launch on iOS 12 and earlier.

<hr /><em>This discussion was created from the release <a href='https://github.com/zbrateam/Zebra/releases/tag/v1.1.28'>v1.1.28</a>.</em>

**Comment by SteveZMTstudios** [comment]
I have a suggestion.
Maybe we can add an “All software Sources”?
This will make user surf the list of all packages easily.
Thanks!
